package.json

```diff
@@ -25,7 +25,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.3.1",
-    "html-parse-stringify2": "2.0.1"
+    "html-parse-stringify2": "2.0.1",
+    "react-test-renderer": "^17.0.1"
```

Comment:
shouldn't react-test-renderer be a devDependency?

Reply:
Yes. Our mistake

Reply:
@tigerabrodi can you move to dev deps, please. 😄